README.md

```diff
@@ -134,6 +143,49 @@ configuration to the virtual machines. The deployment uses the built in default
 networking configuration that Vagrant instantiates. At the end of the run, the
 web interface addresses for Jenkins and Kibana will be displayed.
 
+## Base Deployment (docker)
+
+Start by creating `hosts/containers` (or similar) and add your baremetal machine
+with the following template:
+
+    jenkins_master
+    logstash
+    elasticsearch
+    kibana
+
+These name (e.g. jenkins_master, logstash...) should be matched with container name in cira-container.yml.
```

Comment:
These names (e.g. jenkins_master, logstash, etc.) should match the names as defined in `cira-container.yml`.